File: Baekjoon/1065/1065.py
input_num = input()
if len(input_num) < 3:
    print(input_num)
else:
    result = 99
    for i in range(100, int(input_num) + 1):
        if int(str(i)[0]) + int(str(i)[2]) == 2 * int(str(i)[1]):
            result = result + 1
    print(result)

File: Baekjoon/1065/1065.cc
#include <iostream>

using namespace std;

int main(void) {
    int input_num = 0;
    cin >> input_num;
    if (input_num < 100) {
        cout << input_num << '\n';
    } else {
        int result = 99;
        for (int i = 100; i <= input_num; i++) {
            if ( (i/100 + i%10) == 2*((i/10) % 10)) {
                result++;
            }
        }
        cout << result << '\n';
    }
    return 0;
}